kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -1239,35 +1679,34 @@ namespace Testing {
                 }
             }
         }
-        // Cuts directly through a node are not considered
-        KRATOS_CHECK_EQUAL(n_intersected, 0);
-        KRATOS_CHECK_EQUAL(n_incised, 4);
+
+        KRATOS_CHECK_EQUAL(n_intersected, 4);
+        KRATOS_CHECK_EQUAL(n_incised, 2);
```

Comment:
This test corresponds to (after applying an epislon):

![image](https://user-images.githubusercontent.com/32136457/116856256-1e64de00-abfb-11eb-9e38-a37e9dbfb5a1.png)

Now elem 3 and 6 are incised (n_incised = 6) and elemns 4, 1, 2, 5 are intersected (n_intersected = 4).